src/librustc_codegen_llvm/back/linker.rs

```diff
@@ -912,11 +900,9 @@ fn exported_symbols(tcx: TyCtxt, crate_type: CrateType) -> Vec<String> {
         // For each dependency that we are linking to statically ...
         if *dep_format == Linkage::Static {
             // ... we add its symbol list to our export list.
-            for &(symbol, level) in tcx.exported_symbols(cnum).iter() {
-                if level.is_below_threshold(export_threshold) {
-                    symbols.push(symbol.symbol_name(tcx).to_string());
-                }
-            }
+            symbols.extend(tcx.exported_symbols(cnum).iter()
```

Comment:
Same as above it seems like this wouldn't actually provide any benefit due to the filter